docs/src/tutorials/my-first-notebook/conclusion.md

```diff
@@ -17,5 +17,5 @@ __Where next?__
 You may want to consider these next steps:
 
 * To learn more about JupyterLab, visit <https://jupyterlab.readthedocs.io/en/stable/>.
-* To learn how to manage your Jupyter Python environment using Conda, visit the
-[Conda Environments Quick-Start-Guide](../conda_environments.md).
+* To read on how to get set up with a Python envionrment with Jupyter, visit the
+[Conda Environments Quick-Start-Guide](../getting-started-jupyter/README.md).
```

Comment:
The link title needs updating, and this link doesn't work - it goes to http://localhost:3000/tutorials/getting-started-jupyter/README.html instead of http://localhost:3000/tutorials/getting-started-jupyter/index.html.